orbitdeterminator/kep_determination/ellipse_fit.py

```diff
@@ -385,7 +349,18 @@ def plot_kep(kep,data):
 
     fig = plt.figure()
     ax = Axes3D(fig)
-    ax.axis('equal')
+    try:
+      ax.set_aspect('equal')
+    except NotImplementedError:
+      print("Warning: The equal aspect ratio of 3d plots in matplotlib is currently an issue within the library.")
```

Comment:
You should neither try to set the aspect nor print this warning.

Reply:
hey @Allan-Perez and @tacaswell how shall we proceed now?